### DISCUSSION TITLE
custom search result card gallery

### DISCUSSION THREAD

**Discussion by captainclito** [microsoft-search/pnp-modern-search#2038] [Q&A] [answered]
Hello,

i want to show the modifiedby field instead of author in the search result card gallery but each time i try to do it it doesn't display anything. 
To do this 
1) i select modifiedby properties on selected properties of page 1
2) i select card on page 2 then i'm stuck i don't really know how to continue cause i don't konw if i should change type or result or model of result (i try to modify it but it doesn't work);

do you have any idea?

**Comment by wobba** [comment, marked as answer]
1. The managed property for last modifier is `EditorOWSUser` and you need to do a string split to get the name from the property as it included e-mail and claim as well.

2. Select the card, then click the manage card fields to enter expressions for the property you want to use.

Hope this helps. See https://microsoft-search.github.io/pnp-modern-search/usage/search-results/layouts/

**Reply by captainclito** [reply]
thanks @wobba . actually i didn't have to split aything. what i did i selecting EditorOWSUSER and ModifiedOWSDATE as ModifiebBy and ModifiedByDate.then in manage card fields i enter directly {{getDate (slot item @root.slots.ModifiedDate) 'LL'}} and {{slot item @root.slots.ModifiedBy}}

**Reply by wobba** [reply]
Smart as we parse that internally :)
